core/field_colour.js

```diff
@@ -176,8 +177,11 @@ Blockly.FieldColour.prototype.render_ = function() {
  * @protected
  */
 Blockly.FieldColour.prototype.doClassValidation_ = function(newValue) {
-  if (Blockly.FieldColour.COLOUR_REGEX.test(newValue)) {
```

Comment:
FYI, this changes leaves `Blockly.FieldColour.COLOUR_REGEX` orphaned.  Don't worry about it, I'm removing it in a PR I'm working on.

Reply:
Gah! Thank you for noticing!